plugins/Marketplace/API.php

```diff
@@ -76,8 +76,6 @@ public function __construct(
     }
 
     /**
```

Comment:
Should be at least have the single param or not because it’s explicit?
```suggestion
    /**
     * @param string $email
```

Reply:
Explicitly typed params don't need docblock annotations if they don't have extra information added to it